lib/main_common.pm

```diff
@@ -2351,6 +2351,7 @@ sub load_system_prepare_tests {
     }
     loadtest 'console/integration_services' if is_hyperv || is_vmware;
     loadtest 'console/hostname' unless is_bridged_networking;
+    loadtest 'kernel/install_kernel_flavor' if get_var('KERNEL_FLAVOR');
```

Comment:
I'm aware about similar approach for RT. RT will be redone and migrated to `KERNEL_FLAVOR` as well in separate change.

Reply:
RT part will be migrated  in https://progress.opensuse.org/issues/132542